people_tool/templates/people_tool/index.html

```diff
@@ -43,7 +43,7 @@
     <script src="{% static 'course_info/js/polyfills/autofill-event.js' %}"></script>
     {% include 'django_auth_lti/resource_link_id.html' %}
 
-    <title>Find People</title>
+    <title>Search People</title>
```

Comment:
Looks like the `course_info/templates/course_info/index.html`'s `<title>Course Info</title>` needs updating too.